tracing-error/src/lib.rs

```diff
@@ -65,8 +74,71 @@
 //!     }
 //! }
 //! ```
-//! In the future, this crate may also provide its own `Error` types as well,
-//! for users who do not wish to use other error-handling libraries.
+//!
+//! This crate also provides the [`InstrumentResult`] and [`InstrumentError`]
+//! traits, which can be used to wrap errors with a [`TracedError`] which
+//! bundles the inner error with a [`SpanTrace`].
+//!
+//! ```rust
+//! # use std::error::Error;
+//! use tracing_error::prelude::*;
+//!
+//! # fn fake_main() -> Result<(), Box<dyn Error>> {
+//! std::fs::read_to_string("myfile.txt").in_current_span()?;
+//! # Ok(())
+//! # }
+//! ```
+//!
+//! Once an error has been wrapped with with a [`TracedError`] the [`SpanTrace`]
+//! can be extracted one of 3 ways, either via [`TracedError`]'s
```

Comment:
```suggestion
//! can be extracted one of 3 ways: either via [`TracedError`]'s
```

Reply:
also, prefer to spell out numbers < 10 in text:
```suggestion
//! can be extracted one of three ways, either via [`TracedError`]'s
```